README.md

```diff
@@ -1 +1,13 @@
-# bitrise-add-new-project
\ No newline at end of file
+# bitrise-add-new-project
+
+Creates new project on Bitrise.io using a local existing bitrise.yml or running the Bitrise scanner.
+
+## one-liner
```

Comment:
Pls update the readme to be more user-friendly. For example mention that this command can be run from a bash terminal, on osx and ubuntu